.github/workflows/manual-trigger.yml

```diff
@@ -0,0 +1,25 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual workflow
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      name:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: 'Person to greet'
+        # Default value if no value is explicitly provided
+        default: 'Hello world'
+        # Input is optional for the workflow to run
+        required: false
+
+jobs:
+  greet:
+    runs-on: ubuntu-latest
+    steps:
+    # Runs a single command using the runners shell
+    - name: Send greeting
+      run: echo "Hello ${{ github.event.inputs.name }}"
```

Comment:
[online resource](https://levelup.gitconnected.com/how-to-manually-trigger-a-github-actions-workflow-4712542f1960), fun to test:
> You’ll also notice that you can customise the branch to run your manual workflow against, however be advised that at least some variation of the workflow file must be on your default branch (e.g master) for it to appear in the “Actions” tab.
Selecting a custom branch will:
Checkout the latest commit from that branch.
Use the branch’s own variation of the workflow file (if modified).

Reply:
https://github.com/EQWorks/release/runs/1442665326?check_suite_focus=true

Reply:
`workflow_dispatch.input` seems very nice